saleor/graphql/checkout/types.py

```diff
@@ -154,12 +159,28 @@ def is_shipping_required(product_type):
         )
 
 
+class DeliveryMethod(graphene.Union):
+    class Meta:
```

Comment:
We should a description that will explain what this union represents.

Reply:
Done.